tests/MarginCalculatorTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace PivotLibre\Tideman;
+
+use PHPUnit\Framework\TestCase;
+
+class MarginCalculatorTest extends TestCase
+{
+    public function testMarginCalculator() : void
```

Comment:
There are no tests on this class yet because the design of the class is still pretty uncertain. I would like to flesh these tests out after we figure out a better idea for the implementation.